src/Cutscenes.ts

```diff
@@ -1571,6 +1695,115 @@ export class Cutscenes<TEightBittr extends FullScreenPokemon> extends EightBittr
     }
 
     /**
+     *  Cutscene for a Surf attack in battle.
+     * 
+     * @param settings   Settings used for the cutscene.
+     * @param args   Settings for the routine.
+     */
+    public cutsceneBattleAttackSurf(settings: IBattleCutsceneSettings, args: IBattleAttackRoutineSettings): void {
+        const defenderName: string = args.defenderName;
+        const defender: IThing = this.EightBitter.BattleMover.getThing(defenderName) as IThing;
+        const dt: number = 1;
+        const direction: number = defenderName === "opponent" ? -1 : 1;
+        const differenceY: number = defender.height / 3 * this.EightBitter.unitsize;
+        const lineArray: IThing [] = [];
+        const menu: IMenu = this.EightBitter.MenuGrapher.getMenu("BattleDisplayInitial") as IMenu;
+        const wave1: IThing = this.EightBitter.ObjectMaker.make("ExplosionSmall");
+        const wave2: IThing = this.EightBitter.ObjectMaker.make("ExplosionSmall");
+        let dropplets: IThing[] = [this.EightBitter.ObjectMaker.make("ExplosionSmall")];
+        for (let drops: number = 0; drops < 48; drops++) {
```

Comment:
> `drops++`

Standard here is `+= 1`, to maintain consistency and reduce probability of typos.